cms/static/js/views/settings/grading.js

```diff
@@ -298,10 +303,14 @@ var GradingView = ValidatingView.extend({
         this.descendingCutoffs.push({designation: this.GRADES[gradeLength], cutoff: failBarWidth});
         this.descendingCutoffs[gradeLength - 1]['cutoff'] = Math.round(targetWidth);
 
-        var $newGradeBar = this.gradeCutoffTemplate({ descriptor : this.GRADES[gradeLength],
-            width : targetWidth, removable : true });
+        var newGradeHtmlString = this.gradeCutoffTemplate({
+            descriptor : this.GRADES[gradeLength],
+            width : targetWidth,
+            contenteditable: true,
+            removable : true }).toString();
         var gradeDom = this.$el.find('.grades');
-        gradeDom.children().last().before($newGradeBar);
+
+        gradeDom.children().last().before(newGradeHtmlString);
```

Comment:
@awaisdar001: FYI: I'm trying to add a new call to the UI Toolkit, but it may not be until Tuesday due to our holiday.  Ultimately, I think this line will look like:
```
gradeDom.children().last().before(HtmlUtils.forJQuery(newGradeHtml));
```
Also, the `toString()` call will need to be dropped when setting up newGradeHtml().  If you need to get this PR closed before the new helper is available, then I would move the `toString()` so you have:
```
gradeDom.children().last().before(newGradeHtml.toString());
```
You could then leave this particular change to a future simple PR.  That's up to you.

Thank you.